### ISSUE TITLE
Basic example: Connection timeout on geoportal.sovzond.ru

### ISSUE THREAD

**Issue by BobbyBabes** [bambrikii/leaflet-layer-tree-plugin#4]
I'm trying to get the basic example up and running but eventually am presented with a timeout error. See screenshot 1.

Right-clicking the URL in the error message, followed by `Open in new tab` displays an empty tab with `Connecting...` in the status bar for a few minutes. But then times out and displays the following message. See screenshot 2.
```This site can't be reached
geoportal.sovzond.ru took too long to respond

ERR_CONNECTION_TIMED_OUT
```

**EDIT:** The `Preview:` link for the basic example in the README has https vs http request issues.
So basically no way to find out if this is what I'm after.

![leaflet-layer-tree-plugin-error-01_screenshot_20180513_153435](https://user-images.githubusercontent.com/22870070/39968103-11779e3e-56c8-11e8-8708-2f5cd95cac8e.png)

![leaflet-layer-tree-plugin-error-03_screenshot_20180513_154731](https://user-images.githubusercontent.com/22870070/39968104-183b821c-56c8-11e8-8af6-c7a9554e5197.png)

**Comment by bambrikii** [comment]
updated the examples page. some valid server and layers are used. also for Chrome CORS plugin should be used to avoid cross origin restrictions ![image](https://user-images.githubusercontent.com/585930/40588760-a44bd2b4-61b0-11e8-9288-f49e99cc6ede.png)
